packages/theme-a11y/theme-a11y.js

```diff
@@ -102,14 +138,16 @@ export function focusable(container) {
  *
  * @param {Element} container - Container DOM element to trap focus inside of
  * @param {Element} elementToFocus - Element to be focused on first
+ * @param {Object} options - Settings unique to your theme
+ * @param {string} options.className - Class name to apply to element on focus.
```

Comment:
Since `options` is an object containing properties, can you separate the documentation?